src/elements/input/input.svelte

```diff
@@ -197,33 +213,48 @@ const handleNumberDragDown = async (event: PointerEvent) => {
   window.addEventListener('pointermove', handleNumberDragMove);
   window.addEventListener('pointerup', handleNumberDragUp, { once: true });
 };
-
 </script>
 
+<style>
```

Comment:
Oof, I don't like this one 😭 , because it's inconsistent with the svelte docs / convention of putting style last. I know prettier is opinionated, but is there any way to change?

Reply:
Oh sure, the default for prettier-plugin-svelte is, in fact, to put style last. I only changed it since I saw style in a different place in various files so thought it might be Viam convention.

I'll remove the override which will re-order `style` to be last.

Reply:
Yeah I would like to follow svelte convention here.